### PR TITLE
figure2 edits

### PR DESCRIPTION
**Pull request by MikeLippincott**
This PR making edits to figure 2 addressing comments from @gwaybio.

### CODE REVIEW

File: figures/2.figure2/figures/figure2.png
Comment (jenna-tomkinson) [inline figures/2.figure2/figures/figure2.png:1]: Some comments I have:

Panel A - 
1. Since you label the boxes in the second panel on the same line, maybe you should also label the box in this panel also. But, it might make the plot too busy, but just a suggestion since for me, I immediately drew my eyes to panel B before panel A since I could understand what each cluster represented.

Panel B -
1. This is more of a nit-pick, but I find that it is confusing that the apoptosis box and the control box have very similar colored dots. I can't really tell the difference between the pinks in the control versus apoptosis. To me, it looks like the Thapsigargin dots are in the control box which doesn't make sense as I remember it being a apoptotic inducer.

Panel C - 
1. Looks great! No comments.

Panel D -
1. The axis titles and legend are a bit stretched and a bit blurry. I would recommend changing this since it is visually significantly different than the rest of the plots.

Overall -
1. This figure is looking really good! Some minor visual adjustments and I think it is ready for the manuscript.
2. This is a suggestion that I just thought of, I think it might be useful to have a small figure or table that separates these treatments in the groups that you mention in panel B prior to that panel. I think it is hard for someone to have to look at the colors of the dots and try to make sure they connect the treatments to the right `cell death group` (or however you want to call it). But, you might already talk about this in paragraph form instead, which is valid. I just find that for me and the context switching I go through, it would be nice to have a legend that would orient me to remember which treatment is meant to do what.

File: figures/2.figure2/figures/selected_cytokines_heatmap.png
Comment (jenna-tomkinson) [inline figures/2.figure2/figures/selected_cytokines_heatmap.png:1]: Related to the comments above for panel D, it looks like this figure looks good with the font! It is interesting how adding it to figure 2 makes it so wonky!

File: 2.Nomic_nELISA_Analysis/0.Exploratory_Analysis/PBMC/1.finding_ground_truth.ipynb
Comment (jenna-tomkinson) [inline 2.Nomic_nELISA_Analysis/0.Exploratory_Analysis/PBMC/1.finding_ground_truth.ipynb:1]: Just as an FYI, I would make sure that this notebook is nbconverted to a script before merging.

**Review by jenna-tomkinson — APPROVED** [review]
This is a great figure! 👏 I left comments on the figure itself for you to address prior to merging. I will set this as approved, but if you would like for me to give this PR another look before merging, just let me know!

**Comment by MikeLippincott** [comment]
Thank you for the feedback. I went ahead and made these changes and am ready to merge!

**Pull request merged**
